src/util/parse_options.cpp

```diff
@@ -76,6 +76,12 @@ int parse_options_baset::main()
     std::cerr << e.what() << "\n";
     return CPROVER_EXIT_USAGE_ERROR;
   }
+  catch(system_exceptiont &e)
```

Comment:
We should probably add a base class like `cprover_exceptiont` so we don't have to do this for every single kind of exception we introduce.